proxy/tcpproxy/userspace.go

```diff
@@ -33,15 +39,17 @@ func (r *remote) inactivate() {
 	r.inactive = true
 }
 
-func (r *remote) tryReactivate() {
+func (r *remote) tryReactivate(it time.Duration) {
```

Comment:
probably we should return error from tryReactive and print out everything in monitor function. it is unclear why this function needs an additional it argument.